parser/statement_parsing.go

```diff
@@ -170,7 +193,11 @@ func (p *Parser) parseReturnStatement() *ast.ReturnStatement {
 func (p *Parser) parseExpressionStatement() *ast.ExpressionStatement {
 	stmt := &ast.ExpressionStatement{Token: p.curToken}
 
-	stmt.Expression = p.parseExpression(LOWEST)
+	precedence := 1
```

Comment:
@st0012 
 I will add a new constants to express precedence which is call_without_parens case.
But I just name new constants "FIRST" which is lower than "LOWEST" constant.
Is it appropriate to name it "FIRST" before "LOWER"?
or I just change current "LOWEST" to be "LOWER" or something
and make my new constants be "LOWEST"?